eng/common/TestResources/New-TestResources.ps1

```diff
@@ -45,6 +45,10 @@ param (
     [ValidatePattern('^[0-9a-f]{8}(-[0-9a-f]{4}){3}-[0-9a-f]{12}$')]
     [string] $ProvisionerApplicationId,
 
+    [Parameter(ParameterSetName = 'Provisioner', Mandatory = $false)]
+    [ValidatePattern('^[0-9a-f]{8}(-[0-9a-f]{4}){3}-[0-9a-f]{12}$')]
+    [string] $ProvisionerApplicationOid,
```

Comment:
Do we have this already in the sub-config?

Reply:
We have the value (via `TestApplicationOid` which is the same as provisioner for CI), but the key is not set. I set the parameter mandatory as `$false` so it will still be backwards compatible, and we can then update the sub configs (starting with the dogfood related ones).